pages/details-page/index.js

```diff
@@ -0,0 +1,14 @@
+import Head from 'next/head'
+import Layout from '../../components/layout/layout'
+import styles from './Index.module.css'
+
+export default function DetailsPage() {
```

Comment:
This can be renamed to Details. The filename too.

Reply:
There is already a component by the name Details

Reply:
Ah ok then. Could you rename just the filename then? That way the url will be `yourwebsite.com/details` instead of `yourwebsite.com/details-page`